docs/more/community-projects.md

```diff
@@ -0,0 +1,16 @@
+# Projects
+
+Here are some community projects related to `asdf`:
+
+- [asdf-community](https://github.com/asdf-community): A collaborative,
+  community-driven project for long-term maintenance of asdf plugins.
+- [asdf dev container](https://github.com/iloveitaly/asdf-devcontainer): A
+  [GitHub Dev Container](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers)
+  supporting asdf managed tools in GitHub Codespaces.
+
+::: warning Note
+
+asdf core do not own these projects or their code. asdf core are not responsible
```

Comment:
It might be good to indicate this is about the team of people who work on core. Maybe "asdf core **team** do not own these..."